types/jasmine/jasmine-tests.ts

```diff
@@ -2124,29 +2124,35 @@ describe("version", () => {
     afterAll(function() {
         const jsApiReporter: jasmine.JsApiReporter = (window as any).jsApiReporter;
         const suites = jsApiReporter.suites();
+        const time = jsApiReporter.executionTime();
+
+        console.log("time", time);
 
         for (const k in suites) {
-            const suite: jasmine.SuiteResult = suites[k];
+            const suite: jasmine.CustomReporterResult = suites[k];
             console.log(suite);
             console.log("id", suite.id);
             console.log("description", suite.description);
             console.log("fullName", suite.fullName);
             console.log("fe", suite.failedExpectations);
 
-            for (const fe of suite.failedExpectations) {
-                console.log(">> matcherName:", fe.matcherName);
-                console.log(">> passed:", fe.passed);
-                console.log(">> expected:", fe.expected);
-                console.log(">> actual:", fe.actual);
-                console.log(">> message:", fe.message);
-                console.log(">> stack:", fe.stack);
+            if (suite.failedExpectations) {
+              for (const fe of suite.failedExpectations) {
+                  console.log(">> matcherName:", fe.matcherName);
+                  console.log(">> passed:", fe.passed);
+                  console.log(">> expected:", fe.expected);
+                  console.log(">> actual:", fe.actual);
+                  console.log(">> message:", fe.message);
+                  console.log(">> stack:", fe.stack);
+              }
             }
 
             console.log("dw", suite.deprecationWarnings);
 
-            for (const fe of suite.deprecationWarnings) {
-                console.log(">> message:", fe.message);
-                console.log(">> stack:", fe.stack);
+            if (suite.deprecationWarnings) {
```

Comment:
Why surround this and L2139 in an if?

Reply:
`suite.deprecationWarnings` can be undefined